packages/@okta/vuepress-site/docs/guides/oie-upgrade-mfa-enroll-policy/main/index.md

```diff
@@ -28,6 +28,8 @@ The following are the main behavior changes to the [authenticator enrollment pol
 - Existing authenticator enrollment policies (that is, policies created before the Identity Engine upgrade) still contain factors in their settings.
 - For authenticator enrollment policies modified from the Admin Console of an Identity Engine org, existing factors are converted to authenticators in their settings.
 
+> **Note:** For Identity Engine orgs with the Authenticator enrollment policy feature enabled, the new default authenticator enrollment policy created by Okta uses the authenticators setting schema. Existing default authenticator enrollment policies, prior to enabling this feature, remains unchanged and still uses the factors setting schema. If you want the default policy to use the authenticators setting schema, then modify the default policy from the Admin Console to convert the settings from factors to authenticators.
```

Comment:
Comments apply to below as well:
- Since default authenticator enrollment policy is created by enabling Authenticator enrollment policy feature, so there's no default authenticator enrollment policy prior to enabling this feature. So eventually only migrating orgs will have default enrollment policy in factor format.